app/internal/windows/windows.go

```diff
@@ -814,3 +926,124 @@ func (p *WindowPlacement) Set(Left, Top, Right, Bottom int) {
 	p.rcNormalPosition.Right = int32(Right)
 	p.rcNormalPosition.Bottom = int32(Bottom)
 }
+
+func RegisteredSchemes(appid string) []string {
```

Comment:
These functions doesn't seem to belong here, a package for Windows syscalls. Why not have them in package `app`?